dist/angular-leaflet-directive.js

```diff
@@ -2848,6 +2884,35 @@ angular.module("leaflet-directive").directive('controls', ["$log", "leafletHelpe
                     }
                 }
 
+                if(isDefined(controls.minimap)) {
+                    if (leafletHelpers.MiniMapControlPlugin.isLoaded()) {
+                        if(isDefined(controls.minimap.layer)) {
+                            var layer = createLayer(controls.minimap.layer);
+                            delete controls.minimap.layer;
+
+                            if(isDefined(layer)) {
+                                if(isDefined(leafletScope.center)) {
+                                    var moveend = function(/* event */) {
+                                        var minimapControl = new L.Control.MiniMap(layer, controls.minimap);
+                                        map.addControl(minimapControl);
+                                        map.off('moveend', moveend);
+                                    };
+                                    map.on('moveend', moveend);
+                                } else {
+                                    var minimapControl = new L.Control.MiniMap(layer, controls.minimap);
+                                    map.addControl(minimapControl);
+                                }
+                            } else {
+                                $log.warn('[AngularJS - Leaflet] Layer could not be created.');
```

Comment:
The `[AngularJS - Leaflet]` string is in leafletHelpers.errorHeader. You could extend that string locally to `errorHeader + '  Layer'.